pkg/preflight/checkers/checks.go

```diff
@@ -99,9 +100,20 @@ func StartCmdChecks(vm *api.VM, ignoredPreflightErrors sets.String) error {
 	for _, dependency := range constants.PathDependencies {
 		checks = append(checks, ExistingFileChecker{filePath: dependency})
 	}
+	// This should be warning only, in case users prefer other network plugins
 	if providers.NetworkPluginName == network.PluginCNI {
-		for _, dependency := range constants.CNIDependencies {
-			checks = append(checks, ExistingFileChecker{filePath: dependency})
+		// In case of using ignitew, we put CNI plugin binaries relatively to the executable
+		exeDir := filepath.Dir(os.Args[0])
+		if _, err := os.Stat(exeDir); err == nil {
```

Comment:
I believe this may always be true -- it doesn't indicate that the binary is actually `ignitew`

Perhaps there's a warp native way to detect that the binaries have been unpacked for us